script/quick_test

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
Can we add comments and/or docs about how to use this? (how to call it, what are the expected results?)

Reply:
yep can do